src/main/java/duke/tasks/TaskType.java

```diff
@@ -0,0 +1,8 @@
+package duke.tasks;
+
+/**
+ * Enum class to define the types of Task objects to manage.
+ */
+public enum TaskType {
+    TODO, DEADLINE, EVENT
+}
```

Comment:
I like how you did the enumerations, similar to declaring static final variables, it makes it easier to amend code in the future and there is a smaller chance of careless errors this way